src/components/Tooltip/styles.css

```diff
@@ -8,7 +8,7 @@
   pointer-events: none;
   z-index: var(--z-index-big);
   background-color: var(--tooltip-background-color);
-  visibility: hidden;
+  visibility: visible;
```

Comment:
This change will make the Tooltip always visible. I think we should stick with hidden, shouldn't we?

Reply:
Sure, reverted !